Dockerfile-stretch

```diff
@@ -0,0 +1,20 @@
+# -*-Dockerfile-*-
+FROM golang:stretch
```

Comment:
Maybe use stretch-slim here?

Reply:
1. there is no golang:stretch-slim.
2. builder pattern is only concerned with size of final product which does use stretch-slim.

Reply:
ahm, so that is stretch with golang already installed. But why would you do this? Only for getting `gosu` which can be installed via `apt install`? Is gosu in stretch not usuable?

Reply:
1. habit - most distros do not include gosu or, _e.g._, with brew it is https://gosu-lang.github.io/, not https://github.com/tianon/gosu.
2. i wrote Dockerfile-alpine first and quickly ported to Dockerfile-stretch.
3. i checked stretch and gosu is indeed tianon, updated Dockerfile-stretch.